en/08_Changelogs/6.0.0.md

```diff
@@ -1360,11 +1391,15 @@ One consequence of this is that [`CMSSiteTreeFilter`](api:SilverStripe\CMS\Contr
 
 The `DNADesign\Elemental\Controllers\ElementSiteTreeFilterSearch` class has also changed - it is now [`ElementalSiteTreeSearchContext`](api:DNADesign\Elemental\ORM\Search\ElementalSiteTreeSearchContext). All of the configuration that applied to the old class applies to the new one, so you may need to update that name in your configuration. Refer to [searching blocks](/optional_features/elemental/searching-blocks/#cms-page-search) for more information about this class and the performance impacts it brings with it.
 
-### `FormField::Value()` split into two methods {#formfield-value}
+### `FormField::Value()` split into three methods {#formfield-value}
+
+`SilverStripe\Forms\FormField::Value()` has been split into three methods:
 
-`SilverStripe\Forms\FormField::Value()` has been split into to [`FormField::getValue()`](api:SilverStripe\Forms\FormField::getValue()) which usually returns an unmodified version of the value, and [`FormField::getFormattedValue()`](api:SilverStripe\Forms\FormField::getFormattedValue()) which is intended to be modified with things like localisation formatting and will be displayed to users.
+- [`FormField::getValue()`](api:SilverStripe\Forms\FormField::getValue()) which usually returns an unmodified version of the value
+- [`FormField::getFormattedValue()`](api:SilverStripe\Forms\FormField::getFormattedValue()) which is intended to be modified with things like localisation formatting and will be displayed to users
+- [`FormField::dataValue()`](api:SilverStripe\Forms\FormField::dataValue()) which represents the value as passed into a record when [`saveInto()`](api:SilverStripe\Forms\FormField::saveInto()) is called. Usually this is the same as `getValue()`.
```

Comment:
`dataValue()` _did_ already exist, but there were a few places where `Value()` had been called before that had to become `dataValue()` in core code - so I'm mentioning it here in case others have similar situations.